### ISSUE TITLE
MinGW build broken since very recent commit

### ISSUE THREAD

**Issue by SMOKE5** [173210/psp2sdk#18]
I had successfully compiled and installed the sdk yesterday on Windows with MinGW. Since then, several commits have been pushed, so I wanted to update everything. It throws an error after an extremely long command. I had talked with tomtomdu80 about this exact same error before but on Cygwin and was not resolved.

Here's the log of the last command ran and the error:
http://pastebin.com/mcabQT5u

**Comment by SMOKE5** [comment]
Windows has a command length limit of 8191 characters for a single command as seen here:
https://support.microsoft.com/en-us/kb/830473

This is the commit that broke it: https://github.com/173210/psp2sdk/commit/64f9ef8407fbed2a72d96865d8812b455765b48b

For now, I will just remove this line in configure.ac and will also create a Linux VM for this
AC_CHECK_TOOL(AR, ar, :)

Thanks psxdev

**Comment by psxdev** [comment]
check first if .a are valid with arm-none-eabi-objdump -D yourfile.a

if you get Malformed archive message you will need to use the AC_CHECK_TOOL(AR, ar, :)

**Comment by SMOKE5** [comment]
Yes the file is fine and doesn't throw any malformed archive message

**Comment by 173210** [comment]
The length of command doesn't matter.
```shell-session
arm-none-eabi-ar cru libstdc++_stub.a _SceLibstdc++-0001_head.o _SceLibstdc++-0001_NIDS.o _SceLibstdc++-0001_F00_*.o
/bin/sh: /c/devkitPro/devkitARM/bin/arm-none-eabi-ar: Bad file number
```

**Comment by 173210** [comment]
Fixed with commit b58e7638498b5e13a38c79ad74db28b86e458bd2.
Check it out and report.

**Comment by SMOKE5** [comment]
yes, that fixes it
